frontend/src/routes/smartgoals/+page.svelte

```diff
@@ -1,30 +1,141 @@
-<div class="card flex-shrink-0 w-full lg:w-1/3 max-w-sm shadow-2xl bg-base-100">
-  <div class="card-body">
-    <div class="form-control">
-      <label class="label" for="email-input">
-        <span class="label-text">Email</span>
-      </label>
-      <input type="text" placeholder="email" id="email-input" class="input input-bordered" />
-    </div>
+<script>
+  import { onMount } from 'svelte';
 
-    <div class="form-control">
-      <label class="label" for="password-input">
-        <span class="label-text">Password</span>
-      </label>
-      <input
-        type="password"
-        placeholder="password"
-        id="password-input"
-        class="input input-bordered"
-      />
-      <a href={'#'} class="label-text-alt link link-hover mt-4 mb-4">Forgot password?</a>
-      <a href={'#'} class="label-text-alt link link-hover"
-        >Are your Goals Smart yet? Sign up here!</a
-      >
-    </div>
-    <div class="form-control mt-6">
-      <button class="btn btn-primary">Login</button>
+  let goals = {
+      future: [
+          { name: 'Goal 1', details: 'S.M.A.R.T details here', date: '01/01/2024', days: '10', time: '10:00 AM', editing: false }
+      ],
+      inProgress: [
+          { name: 'Goal 2', details: 'S.M.A.R.T details here', date: '01/02/2024', days: '15', time: '02:00 PM', editing: false }
+      ],
+      completed: [
+          { name: 'Goal 3', details: 'S.M.A.R.T details here', date: '01/03/2024', days: '20', time: '04:00 PM', editing: false }
+      ]
+  }
+
+  let showModal = false;
+  /**
+     * @type {{ name: any; }}
+     */
```

Comment:
In specifying typescript files we should also define these types and params in the typescript style.

In addition to that change I think most of the linting errors are because of the `any` type rather than being specific.

Reply:
I can change the current "any" type however specifying the typescript files removed all "any" file problems showing up. Let me know if you think we should change or leave as is since it is resolved at the moment.